src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -129,13 +134,17 @@ def create_volume(cmd, client, account_name, pool_name, volume_name, resource_gr
         subnet_rg = resource_parts['resource_group']
 
     # if NFSv4 is specified then the export policy must reflect this
-    # the RP ordinarily only creates a default setting NFSv3. Export
-    # policy is not settable directly on creation in CLI only via the
-    # add export policy subcommand
+    # the RP ordinarily only creates a default setting NFSv3.
     if (protocol_types is not None) and ("NFSv4.1" in protocol_types):
         rules = []
-        export_policy = ExportPolicyRule(rule_index=1, unix_read_only=False, unix_read_write=True, cifs=False,
-                                         nfsv3=False, nfsv41=True, allowed_clients="0.0.0.0/0",
+        if allowed_clients is None:
+            raise Exception("Parameter allowed-clients needs to be set when protocol-type is NFSv4.1")
```

Comment:
Usually we raise `CLIError` instead of `Exception` which is a generic exception.

Reply:
Changed to CLIError